package-testing/android-relay/app/build.gradle.kts

```diff
@@ -0,0 +1,112 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
+
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.jetbrains.kotlin.android)
+
+    id("com.ncorti.ktfmt.gradle") version "0.20.1"
```

Comment:
interesting, didnt know about ktfmt, i've used spotless before

Reply:
Any preference? I do like that spotless also takes care of gradle files.

Reply:
no preference, whatever works :)